Source/WebCore/page/ios/EventHandlerIOS.mm

```diff
@@ -478,7 +478,7 @@ static bool findViewInSubviews(NSView *superview, NSView *target)
 
     CurrentEventScope scope(event);
 
-    event.wasHandled = handleMousePressEvent(currentPlatformMouseEvent());
+    event.wasHandled = handleMousePressEvent(currentPlatformMouseEvent()).value_or(false);
```

Comment:
I think this should look more like this:
event.wasHandled = handleMousePressEvent(currentPlatformMouseEvent()).wasHandled();

Then the result class should use Expected<bool, something> as storage and have a bool wasHandled() method that returns the bool or false.  That'll make the code more readable and make future coders not wonder if they should use .value_or(false) or .value_or(true) and why.